example/README.md

```diff
@@ -8,6 +8,8 @@ $ npm install
 $ elm-make
```

Comment:
I believe this should now be `elm make src/Main.elm`, also references to `elm-make` in the readme